test/unit/core/url-beautifier/search/parser.ts

```diff
@@ -1,7 +1,7 @@
 import { UrlParser } from '../../../../../src/core/url-beautifier/handler';
 import SearchUrlParser from '../../../../../src/core/url-beautifier/search/parser';
-import suite, { refinement } from '../../../_suite';
 import * as utils from '../../../../../src/core/url-beautifier/utils';
+import suite, { refinement } from '../../../_suite';
```

Comment:
What is the reason for this move? and empty row

Reply:
They were not in sorted order before so I just re-ordered them.

Reply:
Can you remove the extra line please

Reply:
I'm sorry what extra line? I didn't add any extra lines, and in terms of spacing there's just the spacing between the imports, the consts, and the suite, which I think is fine.